contracts/modules/capital/SwapOperator.sol

```diff
@@ -25,6 +26,12 @@ import "../../external/enzyme/IEnzymePolicyManager.sol";
 contract SwapOperator {
```

Comment:
SwapOperator should implement the `ISwapOperator` interface. Seems like the interface still has the `transferRequestedETH` function!